lib/tentacat/pulls/review_requests.ex

```diff
@@ -18,11 +18,14 @@ defmodule Tentacat.Pulls.ReviewRequests do
   end
 
   @doc """
-  Create a review request
+  Create a review request.
 
   Review Request body example:
 
-      ["octocat"]
+  ```elixir
+
+  ["octocat"]
```

Comment:
ideally this needs to be:
```suggestion
  %{"reviewers" => ["octocat"]}
```